### DISCUSSION TITLE
broken images

### DISCUSSION THREAD

**Discussion by royce-reed** [Nola-Devs/Nola-Devs-v2#117] [Ideas]
I've noticed a couple broken image links on a couple pages. Is this being addressed? If not or not assigned, id like to create a component that creates a fallback avatar that would replace the dead image for improved UX.

my proposed fix would be to wrap images (of people) in a component that checks the img link is valid, else use a fallback avatar.

never used svelt before 😅 and wouldn't mind getting my beak wet

![image](https://github.com/user-attachments/assets/c7c911f2-47cd-4cd5-b2fe-5c8115c96c6f)

**Comment by codingMustache** [comment]
Feel free to make an issue clone down and hack away
